picard/webservice.py

```diff
@@ -484,30 +486,35 @@ def _find(self, entitytype, handler, kwargs):
             value = QUrl.toPercentEncoding(unicode(value))
             queryargs[str(name)] = value
         path = "/ws/2/%s" % (entitytype)
-        return self.get(host, port, path, handler, queryargs=queryargs)
+        return self.get(host, port, path, handler, queryargs=queryargs,
+                xml=xml, priority=priority, important=important, mblogin=mblogin,
+                cacheloadcontrol=cacheloadcontrol, refresh=refresh)
 
     def find_releases(self, handler, **kwargs):
-        return self._find('release', handler, kwargs)
+        return self._find('release', handler, **kwargs)
 
     def find_tracks(self, handler, **kwargs):
-        return self._find('recording', handler, kwargs)
+        return self._find('recording', handler, **kwargs)
 
     def find_artists(self, handler, **kwargs):
-        return self._find('artist', handler, kwargs)
+        return self._find('artist', handler, **kwargs)
 
-    def _browse(self, entitytype, handler, kwargs, inc=[], priority=False, important=False):
+    def _browse(self, entitytype, handler, inc=[],
+            xml=True, priority=False, important=False, mblogin=False,
```

Comment:
@mineo Thanks for the PR to the branch fr this in my repo. I will look at it more carefully as soon as I have time.

As for this comment - I agree and will change this at the same time as I review your PR.